cmd/frontend/graphqlbackend/search_results.go

```
@@ -1464,7 +1464,7 @@ func (r *searchResolver) determineRepos(ctx context.Context, tr *trace.Trace, st
 // diff and commit searches where more than repoLimit repos need to be searched.
 func alertOnSearchLimit(resultTypes []string, args *search.TextParameters) ([]string, *searchAlert) {
 	limits := searchLimits()
-
+	repos, _ := args.RepoPromise.Get(context.Background())
```

why can we ignore the error?

---

The only error that can come back is a context error which at this point (inside `alertOnSearchLimit`) seemed irrelevant to me.

---

Edit: And, we don't have a context here, so we cannot even get a context error ;-)